AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_BasicTests.py

```diff
@@ -157,7 +165,22 @@ def MaterialEditor_BasicFunctionalityChecks_AllChecksPass():
             Tests.redo_material_asset_color_change,
             material_editor_utils.get_property(document_id, base_color_property_name) == expected_color)
 
-        # 10. Look for errors and asserts.
+        # 10. Change the lighting background displayed behind the material model in the viewport.
+        lighting_background_asset_path = os.path.join(
+            "@gemroot:MaterialEditor@", "Assets", "MaterialEditor", "LightingPresets",
+            "neutral_urban.lightingpreset.azasset")
+        lighting_background_asset = atom_tools_utils.load_lighting_preset(lighting_background_asset_path)
+        Report.result(
+            Tests.viewport_background_changed, lighting_background_asset is True)
```

Comment:
I would prefer to check the updated value after loading, however the get_current_lighting_preset appears to return a pointer to struct which is not likely something we can handle in python.
True return from load_lighting_preset indicates that the path specified returns an AZ::Render::LightingPreset struct
I think this is almost the same as the background successfully updated. If the value is the same as before the load_lighting_preset for some reason we would not know that an update did not occur.

Reply:
ill leave the True check in but ill also add an asset check - I debugged this before and `load_lighting_preset()` returns False if it doesn't change the background so I know that's a valid test

but we can add an asset check too also to make sure the background asset is correct